docs/admin/performance_tuning.md

```diff
@@ -472,12 +472,12 @@ please refer to the [System Deployment: Agent Startup][6] documentation
 section.
 
 [1]: <https://github.com/daos-stack/daos/tree/master/src/cart> (Collective and RPC Transport)
-[2]: <https://github.com/daos-stack/daos/blob/master/doc/admin/installation.md#distribution-packages> (DAOS distribution packages)
-[3]: <https://github.com/daos-stack/daos/blob/master/doc/admin/installation.md#building-daos--dependencies> (DAOS build documentation)
-[4]: <https://github.com/daos-stack/daos/blob/master/doc/admin/deployment.md#server-startup> (DAOS server startup documentation)
+[2]: <https://github.com/daos-stack/daos/blob/master/docs/graphadmin/installation.md#distribution-packages> (DAOS distribution packages)
+[3]: <https://github.com/daos-stack/daos/blob/master/docs/graphadmin/installation.md#building-daos--dependencies> (DAOS build documentation)
+[4]: <https://github.com/daos-stack/daos/blob/master/docs/graphadmin/deployment.md#server-startup> (DAOS server startup documentation)
```

Comment:
This should really use relative path IMO...

Reply:
Updated